README.md

```diff
@@ -1,26 +1,47 @@
-# Vagrant Developer Environment
+# Vagrant Developer VM
+
+## Tools Included
```

Comment:
Anmerkung: ich bin fan von english title capitalisation

see https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html
oder https://headlinecapitalization.com/

Reply:
Den Original Title hab ich wieder hergestellt, der Change war unnötig, sorry

Reply:
passt für mich beides :)